spec/rspec/core/formatters_spec.rb

```diff
@@ -137,6 +138,16 @@ module RSpec::Core::Formatters
           }.to change { loader.formatters.length }
         end
       end
+
+      context "When a custom formatter exists" do
+        before { loader.add Custom::AGeneralFormatter, output }
+
+        it "adds a subclass of that formatter for the same output target" do
+          expect {
+            loader.add Custom::AMoreSpecificFormatter, output
+          }.to change { loader.formatters.length }
```

Comment:
I think I'd prefer this if the classes were declared as local variables / lets inline rather than using ones prepared in another file, especially as for this spec it's completely unnecessary to have an implementation.

e.g.
```Ruby
 a_general_formatter = Struct.new(:output)
 a_more_specific_formatter = Class.new(a_general_formatter)

before { loader.add a_general_formatter, output }

it "adds a subclass of that formatter for the same output target" do
  expect {
    loader.add a_more_specific_formatter, output
  }.to change { loader.formatters.length }
end
```